shims/spark311/src/main/scala/com/nvidia/spark/rapids/shims/spark311/SparkShimServiceProvider.scala

```diff
@@ -20,9 +20,8 @@ import com.nvidia.spark.rapids.{SparkShims, SparkShimVersion}
 
 object SparkShimServiceProvider {
   // keep 3.1.0 snapshot version for now until 3.1.1 is released
```

Comment:
Shouldn't we remove this comment too? or update it now that we dropped support for 3.1.0?

Reply:
Argh, sorry I missed that.  Fixed.